internal/mail/backends/gomail_test.go

```diff
@@ -55,65 +55,98 @@ func TestSendEmail_InvalidPort(t *testing.T) {
 }
 
 func TestSendEmail_Success(t *testing.T) {
-	smtpServer := mail.MockSMTPServer(t)
-	ctx := context.Background()
-	gm := backends.NewGomailMailer(config.Config{
-		FromEmail:    "example@gmail.com",
-		SMTPHost:     mail.Localhost,
-		SMTPPort:     strconv.Itoa(smtpServer.PortNumber()),
-		SMTPUser:     "user",
-		SMTPPassword: "password",
-	})
-	err := gm.SendEmail(ctx, "example2@gmail.com", "subject", "message")
-	require.NoError(t, err)
-	assert.Len(t, smtpServer.Messages(), 1)
+	testCases := []struct {
+		name        string
+		toEmails    []string
+		expectError bool
+	}{
+		{
+			name:        "single",
+			toEmails:    []string{"example2@gmail.com"},
+			expectError: false,
+		},
+		{
+			name:        "multiple",
+			toEmails:    []string{"example2@gmail.com", "example3@gmail.com", "example4@gmail.com"},
+			expectError: false,
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			smtpServer := mail.MockSMTPServer(t)
+			ctx := context.Background()
+			gm := backends.NewGomailMailer(config.Config{
+				FromEmail:    "example@gmail.com",
+				SMTPHost:     mail.Localhost,
+				SMTPPort:     strconv.Itoa(smtpServer.PortNumber()),
+				SMTPUser:     "user",
+				SMTPPassword: "password",
+			})
+			err := gm.SendEmail(ctx, tc.toEmails, "subject", "message")
+			if tc.expectError {
+				assert.Error(t, err)
+				return
+			}
+			require.NoError(t, err)
+			messages := smtpServer.Messages()
+			assert.Len(t, messages, 1)
+		})
+	}
 }
 
 func TestVariousParameters(t *testing.T) {
```

Comment:
```suggestion
func TestSendEmailWithVariousParameters(t *testing.T) {
```
maybe give it a more descriptive name? What exactly do you test with various parameters?